config-windows.md

```diff
@@ -40,15 +40,15 @@ The following parameters can be specified:
 
 * **`shares`** *(uint16, OPTIONAL)* - specifies the relative weight to other containers with CPU shares. The range is from 1 to 10000.
 
-* **`percent`** *(uint, OPTIONAL)* - specifies the percentage of available CPUs usable by the container.
+* **`maximum`** *(uint, OPTIONAL)* - specifies the portion of processor cycles that this container can use as a percentage times 100 . Range is from 1 to 10000.
```

Comment:
FWIW, I agree that for the Windows shares and maximum values, the ranges should be left in the spec. While the values could be followed through to the implementation [in Microsoft/hcsshim](https://github.com/Microsoft/hcsshim/blob/master/interface.go), the ranges are not to change between kernel versions, and not easily mapped back to matching docs without looking at implementation.